book/website/communication/citable/citable-cff.md

```diff
@@ -0,0 +1,153 @@
+(cm-citable-cff)=
+# (WIP) Software Citation with CITATION.cff
+
+The [Citation File Format](https://citation-file-format.github.io) lets you provide citation metadata for software or datasets in plaintext files that are easy to read by both humans and machines.
+To provide this metadata, start by writing a `CITATION.cff` file and include it with your software or dataset.
+A `CITATION.cff` file aggregates the information in a key-value format that can be easily interpreted and updated by humans, and easily parsed and converted with existing tools.
+
+(cm-citable-cff-why)=
+## Why Use `CITATION.cff`?
+
+There are great advantages in using a `CITATION.cff` file for the citation information for your software!
+
+It's *easier for you*: When you host your software source code on GitHub and have a `CITATION.cff` in your repository, you can use the Zenodo-GitHub integration to automatically publish new releases of the software. Zenodo will use the information from `CITATION.cff` and show it together with the publication. You don't have to edit this information manually on Zenodo anymore.
+
+It's *easier for the users of your software*:
+1. They can directly use the citation information from `CITATION.cff` to cite your software.
+2. If your source code is on GitHub, they will show the citation information in the sidebar as a formatted citation, and also in the BibTeX format. Users can copy either, paste it into their manuscripts, and cite your software correctly.
+3. If they use the Zotero reference manager, they can import the citation metadata directly from the `CITATION.cff` file in the GitHub repository to their reference manager.
```

Comment:
Zotero also has an application so maybe better to keep it as it is. What do you think?

Reply:
I wasn't sure of what the user action was to import the metadata. Does one import the CFF file into the Zotero application or use the Zotero connected on the GitHub page to save the repo? (Does that make sense?)

Reply:
It works using the browser connector. I haven't mentioned this here because I thought it'd be too detailed, but can add of course if you think that's helpful?

Reply:
Upon reflection, I think it's fine as is. If you _do_ have documentation elsewhere that shows how this works, it might be helpful to point to it. But we can also add this at a later stage. I'll resolve this thread.